Description: Debianize
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-07-29

--- python-xstatic-angular-gettext-2.1.0.2.orig/xstatic/pkg/angular_gettext/__init__.py
+++ python-xstatic-angular-gettext-2.1.0.2/xstatic/pkg/angular_gettext/__init__.py
@@ -34,9 +34,9 @@ HOMEPAGE = 'https://angular-gettext.rock
 LICENSE = '(same as %s)' % DISPLAY_NAME
 
 from os.path import join, dirname
-BASE_DIR = join(dirname(__file__), 'data')
+#BASE_DIR = join(dirname(__file__), 'data')
 # linux package maintainers just can point to their file locations like this:
-#BASE_DIR = '/usr/share/javascript/jquery'
+BASE_DIR = '/usr/share/javascript/angular.js'
 
 LOCATIONS = {
     # CDN locations (if no public CDN exists, use an empty dict)
